src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -46,7 +46,7 @@ const StyledImg = styled.img`
   border-top-left-radius: ${p => p.theme.borderRadii.md};
   border-top-right-radius: ${p => p.theme.borderRadii.md};
   padding: ${p => p.theme.space.md};
-  object-fit: cover;
+  object-fit: contain;
```

Comment:
Oh nice, now it works with a single image.

Reply:
This has less than optimal impact on other images. Compare https://5ef14b1d39364e8b08e446d4--zendeskgarden.netlify.app/components/avatar (now scaled down). This might be the right solution. But seems like there is a bigger problem with the selected dimensions of the source image.

Reply:
Continuing thread here, I didn't see this comment box earlier. 

I want to help solve this, but I'm unsure how and I need help. The examples in Avatars are sized the way they are because the Components span half the column width. So if we make them squares they will have a bunch of white space above and below the example. The IconButton example is a square and set to `contain` the square in the full span of the column. If the css is set to `cover` the image assumes the full width of the column and becomes cropped. I can make the example a rectangle in proportion in Abstract, but as it scales down the example gets very small. This seemed like a decent solve responsively. How can I help make this right?

Reply:
Ok, just wanted to bring it up. If this is the best solution from source-to-production, then I'm ok with it.